client/src/style/scss/overrides.scss

```diff
@@ -88,7 +88,7 @@ button {
 // Modal -- wider by default, scroll like Trello
 
 .modal-dialog {
-    width: 690px;
+    // width: 690px; // by removing this, size can be set with the "size" attribute of the bootstrap component <b-modal>
```

Comment:
This has to be reviewed by someone who knows why this property was set in this way.

I removed this because i wanted my modal to be wider than 690px